stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDocumentProvider.java

```diff
@@ -138,11 +141,16 @@ public NodeDescriptor getNodeDescriptor(Object element) {
   public void highlightElement(Object element, int color) {
     verifyThreadAccess();
 
-    View highlightingView = getHighlightingView(element);
-    if (highlightingView == null) {
+    HighlightableDescriptor highlightableDescriptor = getHighlightableDescriptor(element);
+    if (highlightableDescriptor == null) {
       mHighlighter.clearHighlight();
     } else {
-      mHighlighter.setHighlightedView(highlightingView, color);
+      mHighlightBoundsRect.setEmpty();
+      highlightableDescriptor.getBoundsForHighlighting(element, mHighlightBoundsRect);
```

Comment:
Good call! I used to have the `Rect` as a return value which is why there were initially split into two methods. I'll post an update later today. Thanks.